packages/cli/src/cli.ts

```diff
@@ -85,8 +78,8 @@ export async function cli(args: string[] = process.argv) {
     return await base.parseAsync();
   } catch (err) {
     if (err instanceof ExitProcessError) {
-      const Consola = await import("consola");
-      Consola.consola.error(err);
+      consola.error(err);
+      consola.debug(err.stack);
```

Comment:
This is logging the stack as well so need for the second statement. Also the second statement won't be logged as consola level will not be set because this try/catch is for handling errors prior to parsing.
```suggestion
      consola.error(err);
```